testing/kfctl/endpoint_ready_test.py

```diff
@@ -26,7 +26,7 @@ def test_endpoint_is_ready(project, app_name):
   os.environ["CLIENT_ID"] = "29647740582-7meo6c7a9a76jvg54j0g2lv8lrsb4l8g.apps.googleusercontent.com"
   if not gcp_util.endpoint_is_ready(
```

Comment:
Question: Where is code for gcp_util?

Reply:
Sorry I found the code for the same:
https://github.com/kubeflow/kubeflow/blob/fa8886eae006dee0270bcc8bc1f5efc9e3c82e48/testing/gcp_util.py#L78